### DISCUSSION TITLE
Convert pixart alpha model to diffuser error

### DISCUSSION THREAD

**Discussion by segalinc** [PixArt-alpha/PixArt-alpha#67] [Q&A]
I am trying to convert a trained model to diffusers but I when running `convert_pixart_alpha_to_diffusers.py` with same args as guided in the notebook I get the error
```
RuntimeError: Error(s) in loading state_dict for Transformer2DModel:
        Unexpected key(s) in state_dict: "caption_projection.y_embedding". 
```

**Comment by lawrence-cj** [comment]
Right, will upload a new version. This error is caused by the update of `diffusers` lib.

**Comment by segalinc** [comment]
For now I fixed switching the strict arg to false when loading the
convertes dict in the transformer

**Reply by lawrence-cj** [reply]
Already fixed, plz check it out. https://github.com/PixArt-alpha/PixArt-alpha/commit/c3b30e5d35895670b1fb15ddf78b502bf07e26f4

**Comment by segalinc** [comment]
Great thanks
